### DISCUSSION TITLE
不稳定版本导致无法构建

### DISCUSSION THREAD

**Discussion by Shiiyuko** [simple-robot/Discussions#186] [Q&A] [answered]
### 涉及的编程语言

Kotlin

### 涉及的组件库

QQ频道

### 疑问描述

报错：`Class 'love.forte.simbot.qguild.stdlib.Bot.Ticket' is compiled by an unstable version of the Kotlin compiler and cannot be loaded by this compiler.`

Gradle Plugin:
`kotlin("jvm") version "1.9.22`

依赖：
`implementation("love.forte.simbot.component:simbot-component-qq-guild-stdlib:4.0.0-dev8")`

**Comment by ForteScarlet** [comment, marked as answer]
看样子是之前尝试性开启 K2 后忘了关了...
关注 issue https://github.com/simple-robot/simbot-component-qq-guild/issues/123 ，
~~会稍后发布一个新的版本并暂时关闭 K2 编译~~

[v4.0.0-beta2](https://github.com/simple-robot/simbot-component-qq-guild/releases/tag/v4.0.0-beta2) 已发布

当然，另一种解决方式是直接给自己的项目也开启 K2 编译

```kotlin
tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
        kotlinOptions {
            // useK2
            languageVersion = "2.0"
        }
    }
```
